aiohttp_session/nacl_storage.py

```
@@ -27,11 +30,16 @@ def __init__(self, secret_key, *, cookie_name="AIOHTTP_SESSION",
         if cookie is None:
             return Session(None, data=None, new=True, max_age=self.max_age)
         else:
-            data = self._decoder(
-                self._secretbox.decrypt(cookie.encode('utf-8'),
-                                        encoder=Base64Encoder).decode('utf-8')
-            )
-            return Session(None, data=data, new=False, max_age=self.max_age)
+            try:
+                data = self._decoder(
+                    self._secretbox.decrypt(cookie.encode('utf-8'),
+                                            encoder=Base64Encoder).decode('utf-8')
+                )
+                return Session(None, data=data, new=False, max_age=self.max_age)
+            except (binascii.Error, nacl.exceptions.CryptoError):
```

Not sure if we shouldn't just catch all exceptions